packages/wdio-mocha-framework/src/index.js

```diff
@@ -166,16 +166,11 @@ class MochaAdapter {
         if (params.payload) {
             message.title = params.payload.title
             message.parent = params.payload.parent ? params.payload.parent.title : null
-            /**
```

Comment:
this is fine if hook has no parent. If hook has no parent it means that it is global hook and we want to ignore global hooks in report